client/pom.xml

```diff
@@ -20,13 +20,15 @@
       <classifier>bundle</classifier>
       <version>${project.version}</version>
     </dependency>
+    <!--
```

Comment:
If I uncomment Libcloud dep, in my dev env libcloud it is fetched from my local if available and if not from nexus. 
But it also work without libcloud dep. For python api without the dependency it is failing.

For now, I will let it commented as it's working for both of us